### PR TITLE
Added Input

### PR DESCRIPTION
**Pull request by deepak29-git**

### CODE REVIEW

File: component/Card/index.html [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+            <p class="description">
+              Visit ten places on our planet that are undergoing the biggest
+              changes today.
+            </p>
+    
+          </div>
+        </div>
+        <footer class="footer">
+          <div class="icons">
+            <img class="icon" src="./icon/like.png" alt="" />
```
Comment (tanishk15sharma) [inline component/Card/index.html:226]: suggestion: give `alt` messages rest all good

**Review by tanishk15sharma — APPROVED** [review]

File: component/Card/style.css [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+  position: absolute;
+  top: 10px;
+  right: 10px;
+  cursor: pointer;
+  padding: 0.5rem;
+  border-radius:50%;
+}
+
+.badge-icon:hover{
+  background-color: rgb(226, 224, 224);
```
Comment (shivanigangadharan) [inline component/Card/style.css:54]: Instead of rgb or color names, use colour variables. (in all places)

File: component/Image/image.css [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    height:auto;
+    object-fit: cover;
+}
+
+/* round image css */
+
+.round-img{
+    height:9rem;
+    width: 9rem;
+    border-radius:999px;
```
Comment (shivanigangadharan) [inline component/Image/image.css:15]: Can use a css variable here too, like var(--rounded). Will help maintain consistency.

**Review by shivanigangadharan — COMMENTED** [review]
Have suggested changes, you can try that. Rest of the code looks good to me. 👍

File: component/Card/index.html [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+        </div>
+        </div>
+       
+        <footer class="footer">
+          <button class="btn">READ</button>
+          <button class="btn">BOOKMARK</button>
+          <div class="icons">
+            <img class="icon" src="./icon/like.png" alt="" />
+            <img class="icon" src="./icon/share.png" alt="" />
+            <img class="icon" src="./icon/three-dot.png" alt="" />
```
Comment (ttahm3d) [inline component/Card/index.html:94]: You can use font-awesome icons or material Icons.

Font awesome -> 

Use the below css link in head 
```html
 <link
      rel="stylesheet"
      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css"
    />
```

and search for icons here https://fontawesome.com/v5.15/icons

File: component/Card/index.html [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+          <h3 class="title">Our Changing Planet</h3>
+          <p class="small_content">by Kurt Wagner</p>
+          <p class="description">
+            Visit ten places on our planet that are undergoing the biggest
+            changes today.
+          </p>
+        </div>
+        </div>
+       
+        <footer class="footer">
```
Comment (ttahm3d) [inline component/Card/index.html:88]: Dont use Element `<footer >` in middle of the document. There are some HTML tags that have some semantic meaning.  You can read about the meaning of these elements [here](https://www.w3schools.com/html/html5_semantic_elements.asp)

And don't use the class `footer`.

If I'm not wrong, the bottom part has links and buttons. You can wrap them in a div and give a class name as below

```
<div class="card-links-container">
....content
</div>
```

File: component/Card/index.html [hunk 6]
```diff
@@ -0,0 +1,264 @@
...
+        </footer>
+      </div>
+      </div>
+      
+    
+<br>
+<!-- basic with text over media -->
+    <div class="card__container">
+      <div class="card__wrapper text_media">
+        <div class="card_background">
```
Comment (ttahm3d) [inline component/Card/index.html:105]: Follow one style of giving css class names. If you are using double underscore, stick with it

File: component/Card/index.html [hunk 7]
```diff
@@ -0,0 +1,264 @@
...
+          <div class="image_container">
+            <img
+            class="image"
+            src="https://www.planetware.com/wpimages/2020/01/india-in-pictures-beautiful-places-to-photograph-gateway-of-india-mumbai.jpg"
+            alt="img"
```
Comment (ttahm3d) [inline component/Card/index.html:110]: Dont give alt tag value of 'img' or 'image'. It causes accessibility problems.

File: component/Input/input.css [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+    margin: 0;
+    padding: 0;
+}
+.wrapper{
+    display: flex;
+    justify-content: center;
+    margin-bottom: 2rem;
+
+}
+form{
```
Comment (ttahm3d) [inline component/Input/input.css:12]: Dont style the element directly. Add a class to that element and style it

**Review by ttahm3d — COMMENTED** [review]
Have suggested few changes, Please check them.

File: component/Card/index.html (hunk 1 above)
Comment (deepak29-git) [reply component/Card/index.html:226]: ok

File: component/Card/style.css (hunk 2 above)
Comment (deepak29-git) [reply component/Card/style.css:54]: done

File: component/Image/image.css (hunk 3 above)
Comment (deepak29-git) [reply component/Image/image.css:15]: ok

File: component/Card/index.html (hunk 6 above)
Comment (deepak29-git) [reply component/Card/index.html:105]: ok

File: component/Card/index.html (hunk 5 above)
Comment (deepak29-git) [reply component/Card/index.html:88]: ok

File: component/Card/index.html (hunk 7 above)
Comment (deepak29-git) [reply component/Card/index.html:110]: ok

File: component/Input/input.css (hunk 8 above)
Comment (deepak29-git) [reply component/Input/input.css:12]: ok

**Pull request merged**